chapters/hpmor-chapter-061.tex

```diff
@@ -362,6 +362,7 @@
 
 “And they knew about rockets, too?” Severus said dryly. “I don’t believe the other Death Eaters were so fond of Muggle Studies. It is he.”
 
+% sic
```

Comment:
@entorb Would it mess up the ebook scripts if there was a comment (and hence line break) in the middle of a paragraph? I would like to be able to have a `% sic` comment directly after the offending word, so it's obvious what's going on.

Reply:
should not be a problem

Reply:
And again here, please move the `% sic` right after the relevant word.